src/app/zonos/page.tsx

```diff
@@ -74,16 +76,7 @@ const Zonos = () => {
         height: "100vh",
       }}
     >
-      <Script
```

Comment:
This is an old script and it could cause conflicts with the new Zonos stuff. Unnecessary since we have a replacement as part of the other script you are including.